code/Style.md

```diff
@@ -70,6 +72,48 @@ func Example() (err error) {
 }
 ```
 
+Sometimes, there are situations where a function, calls another function that only needs to return the value or the error returned by the call; when the called function return errors not created or wrapped by `errs` package the error must be  wrapped or annotated, however it isn't needed to have a conditional to achieve such purpose, because the functions that `errs` package have ignore the `nil` value when they are passed through; for example
+
+```go
+func OpenDataFile(filename string) (*os.File, error) {
+	file, err := os.Open(filepath.Join("data", filename))
+	if err != nil {
+		return nil, Error.Wrap(err)
+	}
+
+	return file, nil
+}
+```
+
+should be written like
+
+```go
+func OpenDataFile(filename string) (*os.File, error) {
+	file, err := os.Open(filepath.Join("data", filename))
+	return file, errs.Wrap(err)
+}
+```
+
+### Conditionals for _Not Found_ error values
+
+It's pretty common in Go to find functions that return an error when a function should return a value but such value "isn't found". The most classic example of this is the [`sql.ErrNoRows`](https://golang.org/pkg/database/sql/#pkg-variables) which may be returned by function like [`sql#DB.QueryRow`](https://golang.org/pkg/database/sql/#DB.QueryRow). Under this circumstances we first check for such error and if it doesn't match then we do a second check for non `nil` error, rather than nesting them.
+
+```go
+var ErrUser = errs.Class("user")
+
+func (reg *Registry) GetUser(ctx context.Context, username string) (*User, error) {
+	user, := reg.db.Get_User_By_Username(ctx, username)
+	if err == sql.ErrNoRows {
+		return nil, ErrUser.New("not found %s", username)
+	}
+	if err != nil {
+		return nil, Error.Wrap(err)
```

Comment:
What's the reason behind using the generic `Error` here vs `ErrUser`?

Reply:
None, it was forgotten to be changed when copied and pasted.
I fixed it in a new commit.